sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/SampledPointData.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.data;
+
+import java.util.Collection;
+import javax.annotation.concurrent.Immutable;
+
+/** A point that includes (optional) exemplars which lead to its creation. */
+@Immutable
+public interface SampledPointData extends PointData {
```

Comment:
is this necessary/desirable only because SummaryPointData isn't sampled? If so, I think we could just move the method up to PointData itself since the SummaryPointData is legacy anyway. Thoughts?

Reply:
Yeah, we could move it up.  I'm fine making that change, should I wait for more input or just do that now?

Reply:
I'd go ahead and move it up. Makes a code a bit simpler all-in-all.

Reply:
done